gradle-plugin/kubernetes/src/test/java/org/eclipse/jkube/gradle/plugin/task/KubernetesBuildTaskTest.java

```diff
@@ -137,4 +159,43 @@ public Property<Boolean> getSkipBuild() {
     assertThat(dockerBuildServiceMockedConstruction.constructed()).isEmpty();
     verify(buildTask.jKubeServiceHub.getBuildService(), times(0)).build(any());
   }
+
+  @Test
+  @DisplayName("build task invokes generators in expected order")
+  void generatorsAreInvokedInExpectedOrder(@TempDir File temporaryFolder) {
+    // Given
+    extension = new TestKubernetesExtension() {
+      @Override
+      public Property<File> getResourceSourceDirectory() {
+        return super.getResourceSourceDirectory().value(new File(temporaryFolder, "src/main/jkube"));
+      }
+    };
+    extension.javaProject = JavaProject.builder()
+        .baseDirectory(temporaryFolder)
+        .outputDirectory(temporaryFolder.toPath().resolve("target").toFile())
+        .buildDirectory(temporaryFolder.toPath().resolve("target").toFile())
+        .properties(new Properties()).build();
+    when(taskEnvironment.project.getExtensions().getByType(KubernetesExtension.class)).thenReturn(extension);
+    final KubernetesBuildTask buildTask = new KubernetesBuildTask(KubernetesExtension.class);
+    GeneratorContext generatorContext = buildTask.initGeneratorContextBuilder().build();
+
+    // When
+    List<Generator> generatorList = createUsableGeneratorList(generatorContext);
+
+    // Then
+    Assertions.assertThat(generatorList)
```

Comment:
for readability purpose, could you make Assertions.assertThat statically imported ?